src/components/offersInvites/offerInvite/index.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import { Button, Card, Tag } from "antd";
+import {
+    UserOutlined,
+    EnvironmentOutlined,
+    TagOutlined,
+    CalendarOutlined
+} from "@ant-design/icons";
+import moment from 'moment';
+import { offerRoles } from '../../../constants/offerRoles';
+import heavy_icon from "../../../assets/images/cars/heavy.svg";
+import { manageInivite } from "../../../services/invites";
+
+function OfferInvite(data) {
+    const [isAnswered, setIsAnswered] = useState(data.info.isAnswered);
+
+    const onClick = async (isAccepted) => {
+        var answer = await manageInivite({
+            inviteId: data.info.id,
+            isAccepted: isAccepted
+        });
+
+        setIsAnswered(answer);
+    }
+
+    return (
+        <Card className="inviteCard">
+            <h3>My offer :</h3>
+
+            <Card className="innerCard offerCard">
+                <Tag>{data.info.offerInfo.creatorRoleName === offerRoles.SENDER.toUpperCase() ? "Donate" : "Need"}</Tag>
+
+                <p className="description">{data.info.offerInfo.description}</p>
+
+                <div className="information">
+                    <div className="field-group">
+                        <div className="cardField">
+                            <TagOutlined className="fieldIcon" />
+                            <p className="fieldText category">{data.info.offerInfo.goodCategoryName}</p>
+                        </div>
+                    </div>
+
+                    <div className="field-group">
+                        <div className="cardField">
+                            <EnvironmentOutlined className="fieldIcon" />
+                            <p className="fieldText">{data.info.offerInfo.address}, {data.info.offerInfo.settlement}</p>
+                        </div>
+                    </div>
+
+                    <div className="field-group">
+                        <div className="cardField">
+                            <img src={heavy_icon} className="fieldIcon" />
```

Comment:
Let`s use React Icons

Reply:
We've already used this icon in the project